docs/examples/eventrouter.yaml

```diff
@@ -78,11 +105,18 @@ spec:
       template: |
         apiVersion: argoproj.io/v1alpha1
         kind: Workflow
+        metadata:
+          labels:
+            workflows.argoproj.io/controller-instanceid: addon-manager-workflow-controller
```

Comment:
This label is auto-injected btw, not necessary here but this is ok.

Reply:
you are right. this should be removed

Reply:
6fcc8459dc7c95eb1ea7794ac161a8a11d08e1f5